tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -184,3 +192,171 @@ setup_file() {
     run bash -c 'fvm self uninstall --yes'
     assert_success
 }
+
+@test "Copies binaries to Fluvio Binaries Directory when using fvm switch" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Removes Fluvio Directory
+    rm -rf $FLUVIO_HOME_DIR
+
+    # Ensure `~/.fluvio` is not present
+    run bash -c '! test -d $FLUVIO_HOME_DIR'
+    assert_success
+
+    # Installs Stable Fluvio
+    run bash -c 'fvm install stable'
+    assert_success
+
+    # Switch version to use Stable Fluvio
+    run bash -c 'fvm switch stable'
+    assert_success
+
+    # Expected binaries
+    declare -a binaries=(
+        fluvio
+        fluvio-run
+        fluvio-cloud
+        cdk
+        smdk
+    )
+
+    for binary in "${binaries[@]}"
+    do
+        export FVM_BIN_PATH="$VERSIONS_DIR/stable/$binary"
+        echo "Checking binary: $BINARY_PATH"
+        run bash -c 'test -f $FVM_BIN_PATH'
+        assert_success
+
+        export FLV_BIN_PATH="$FLUVIO_BINARIES_DIR/$binary"
+        echo "Checking binary: $FLV_BIN_PATH"
+        run bash -c 'test -f $FLV_BIN_PATH'
+        assert_success
+    done
+
+    # Removes FVM
+    run bash -c 'fvm self uninstall --yes'
+    assert_success
+
+    # Removes Fluvio
+    rm -rf $FLUVIO_HOME_DIR
+    assert_success
+}
+
+@test "Sets the desired Fluvio Version" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Ensure `~/.fluvio` is not present
+    run bash -c '! test -d $FLUVIO_HOME_DIR'
+    assert_success
+
+    # Installs Fluvio at 0.10.15
+    run bash -c 'fvm install 0.10.15'
+    assert_success
+
+    # Installs Fluvio at 0.10.14
+    run bash -c 'fvm install 0.10.14'
+    assert_success
+
+    # Switch version to use Fluvio at 0.10.15
+    run bash -c 'fvm switch 0.10.15'
```

Comment:
Make sure before this, that fluvio is running 0.10.14 by checking version when running